Monika After Story/game/script-topics.rpy

```diff
@@ -15990,3 +15990,106 @@ label monika_stargazing:
     show monika 5kkbsa
     pause 2.0
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_eyesight",
+            category=["you"],
+            prompt="Wearing glasses",
+            random=True
+        )
+    )
+
+default persistent._mas_pm_uses_eyesight_aids = None
+
+label monika_player_eyesight:
+    m 3lsb "Hey, [player], I was wondering about something lately."
+    m 1esa "Do you wear glasses, or maybe contact lenses?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Do you wear glasses, or maybe contact lenses?{fast}"
+
+        "I always wear glasses.":
+            call monika_player_eyesight_awea(glasses=True)
+
+        "I wear contact lenses.":
+            call monika_player_eyesight_awea(glasses=False)
+
+        "I wear cosmetic glasses.":
+            call monika_player_eyesight_cg
+
+        "I wear them when needed.":
+            call monika_player_eyesight_eawn
+
+        "Nope, I don't.":
+            call monika_player_eyesight_n
+
+    return "derandom"
+
+# Always Wears Eyesight Aids
+label monika_player_eyesight_awea(glasses=False):
+    $ persistent._mas_pm_uses_eyesight_aids = True
+    m 2ekc "Oh, I see. You've got poor eyesight, haven't you?"
+    m 2eka "I really hope you see better with them on."
+    if glasses:
+        m 3eub "And there's actually a bright side of wearing glasses - I'm sure you look so smart and intelligent, ehehe~"
+    m 1eka "Please take good care of yourself, [player], even when it comes to something minor like this."
```

Comment:
I don't think eyesight is a minor thing, and I don't think it really fits here to have Monika tell you to take good care of yourself in general. Maybe focus on eyesight.